ui/components/multichain/receive-modal/receive-modal.js

```diff
@@ -28,8 +28,10 @@ import {
 export const ReceiveModal = ({ address, onClose }) => {
   const t = useI18nContext();
   const useBlockie = useSelector(getUseBlockie);
-  const accounts = useSelector(getMetaMaskAccountsOrdered);
-  const { name } = accounts.find((account) => account.address === address);
+  const internalAccount = useSelector((state) =>
+    getInternalAccountByAddress(state, address),
+  );
+  const name = internalAccount?.metadata.name;
```

Comment:
It seems like before name could never be undefined, but here `internalAccount?` could be undefined and then we would not have a name.

Reply:
Yes, it should not be undefined.